src/Playwright.Tests/PageNetworkResponseTests.cs

```diff
@@ -159,7 +162,7 @@ public async Task ShouldWaitUntilResponseCompletes()
 
         class TestClass
```

Comment:
We can drop this class if we are not going to use it for tests.

Reply:
Will follow-up